### ISSUE TITLE
FFMPEG EPERM Error during

### ISSUE THREAD

**Issue by MaximSergeichik** [sitespeedio/browsertime#911]
I'm trying to setup sitespeed.io without docker because of security policies and I faced a problem with the video recording.
I've installed ffmpeg, python and xvfb, configured them but when I'm starting sitespeed.io video can not be recorded because of the following error:

```
Error: Command failed with exit code 1 (EPERM): ffmpeg -hide_banner -video_size 1366x642 -f x11grab -framerate 30 -probesize 10M -y -draw_mouse 0 -i :99.0+0,66 -codec:v libx264rgb -crf 0 -preset ultrafast -vf pad=ceil(iw/2)*2:ceil(ih/2)*2 /tmp/browsertime-gtwA8k/x11.mp4
    at makeError (/home/user/node_modules/sitespeed.io/node_modules/browsertime/node_modules/execa/lib/error.js:58:11)
    at handlePromise (/home/user/node_modules/sitespeed.io/node_modules/browsertime/node_modules/execa/index.js:112:26)
    at process._tickCallback `(internal/process/next_tick.js:68:7)
```

Can you share any solution for this? Or maybe it is possible to make path for this file configurable?

**Comment by soulgalore** [comment]
Hi @MaximSergeichik I’ve seen the error too, but on Travis but I’ve not nailed it down yet (I’m on vacation). Would love to solve ut. What OS are you using? When I looked at it before I got the feeling it was a permission issue for the /tmp/ folder but I’m not sure.

Best 
Peter

**Comment by MaximSergeichik** [comment]
Hi @soulgalore ,

I'm using RHEL7 for this.
May be you know what is the user which is trying to write file in /tmp/browsertime-* folders? I'm not sure that there is the user which is run sitespeed.

I'll try to solve this problem.

Have a nice vac!

Cheers,
Maxim

**Comment by soulgalore** [comment]
Hi @MaximSergeichik I did a fix now where we skip using the OS tmpdir and that fixed things on Travis. We usually use the destination dir as tmpdir and the only place where we used the OS tmpdir was when creating a video. It's merged in master but that is for the coming major (sometimes in September). 

I can backport the fix later next week so you can use it.

Best
Peter

**Comment by MaximSergeichik** [comment]
Hi @soulgalore ,

I'll be really appreciated if you will backport this fix as soon as possible.

Cheers,
Maxim

**Comment by soulgalore** [comment]
Hi @MaximSergeichik I've just released the first alpha of coming 10.0. The tag is 10.0.0-alpha.1 so I think you can use that. To be on the safe side (if you are using Chrome) you should update to Chrome 77 beta. The coming version comes budled with Chromedriver 77 (for Chrome), it may work with 76 but I haven't tried.

`npm install sitespeed.io@10.0.0-alpha.1`

The alpha is pretty stable but ... the stable version of 10 will come the 11th of September.

Best
Peter

**Comment by soulgalore** [comment]
This has been released + one of the issues was missing lib on the server. Closing.
